gudid/Makefile

```diff
@@ -0,0 +1,15 @@
+gudid.sqlite: schema.sql device.txt identifiers.txt contacts.txt gmdnTerms.txt productCodes.txt sterilizationMethodTypes.txt environmentalConditions.txt deviceSizes.txt premarketSubmissions.txt
+	rm gudid.sqlite || true
+	sqlite3 gudid.sqlite < schema.sql
+	mpcsv-read -D '|' < gmdnTerms.txt | mptable2alist -H | mpsqlite --mode=insert gudid.sqlite gmdnTerms > /dev/null
+	mpcsv-read -c -D '|' < device.txt | mptable2alist -H | mpsqlite --mode=insert gudid.sqlite device > /dev/null
+	mpcsv-read -D '|' < productCodes.txt | mptable2alist -H | mpsqlite --mode=insert gudid.sqlite productCodes > /dev/null
+	mpcsv-read -D '|' < identifiers.txt | mptable2alist -H | mpsqlite --mode=insert gudid.sqlite identifiers > /dev/null
+	mpcsv-read -D '|' < sterilizationMethodTypes.txt | mptable2alist -H | mpsqlite --mode=insert gudid.sqlite sterilizationMethodTypes > /dev/null
+	mpcsv-read -D '|' < environmentalConditions.txt | mptable2alist -H | mpsqlite --mode=insert gudid.sqlite environmentalConditions > /dev/null
+	mpcsv-read -D '|' < contacts.txt | mptable2alist -H | mpsqlite --mode=insert gudid.sqlite contacts > /dev/null
+
+# deviceSizes
+# premarketSubmissions
```

Comment:
These look to be left over.

Reply:
Yeah. These are two files that nobody has shown any interest in yet, so I've not added them to the schema and imported them. I've left those comments there as a reminder, but I should elaborate!